Cargo.toml

```
@@ -32,6 +32,9 @@ compile-time-rng = ["const-random"]
 # Enables specilization (requires nightly)
 specialize = []
 
+# Enable serialization and deserialization with serde.
+use-serde = ["serde"]
```

Can we just name this "serde"?
```suggestion
serde = ["serde"]
```

---

I don't think so. I get the following error

```
error: failed to parse manifest at `.../Cargo.toml`

Caused by:
  Features and dependencies cannot have the same name: `serde`
```

We can use [serde1](https://lib.rs/crates/serde1) instead of serde.

---

Or we can do that.

https://github.com/RustCrypto/RSA/pull/41/files

---

Ah, apparently if `serde` is an optional dependency it implies a feature named `serde` hence the conflict. So you can just omit it. (I'm looking at this: https://github.com/rust-lang/hashbrown/blob/34c11891e13fa3c0d08b0540e869aace9d347c26/Cargo.toml) as an example